pallets/oracle/src/lib.rs

```diff
@@ -0,0 +1,442 @@
+//! pallet-oracle queries the ChainLink Price Feed smart contract on Ethereum to fetch the price of FRAG/USD.
+//!
+//! It works as an Offchain Worker that will be triggered after every block, to fetch the current price.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+use frame_support::traits::Get;
+use frame_system::{
+	self as system,
+	offchain::{
+		AppCrypto, CreateSignedTransaction, SendUnsignedTransaction,
+		SignedPayload, Signer, SigningTypes,
+	},
+};
+use sp_core::crypto::KeyTypeId;
+use sp_clamor::http_json_post;
+use serde_json::{json, Value};
+use sp_runtime::{
+	transaction_validity::{InvalidTransaction, TransactionValidity, ValidTransaction},
+	RuntimeDebug,
+};
+use sp_std::vec::Vec;
+
+#[cfg(test)]
+pub mod tests;
+pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"frag");
```

Comment:
shouldn't the keytype be different since this is a differernt pallet (I could be wrong 😅)

Reply:
Oh probably yes.

Reply:
In pallet detach and accounts they are the same.